content/en/opentelemetry/guide/migrate_operation_names.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Migrate to New Operation Name Mappings
+further_reading:
+- link: "/opentelemetry/schema_semantics/"
+  tag: "Documentation"
+  text: "Mapping Datadog and OpenTelemetry Conventions"
+---
+
+## Overview
+
+When using OpenTelemetry with Datadog, you might see unclear or lengthy operation names in your traces, and some traces might not appear in your service pages. This happens because of missing mappings between OpenTelemetry SDK information and Datadog operation names, which are span attributes that classify [entry points into a service][1].
+
+Datadog now provides new operation name mappings that improve trace visibility in the service page. This feature requires opt-in configuration now but will become the default in the near future.
+
+<div class="alert alert-warning">
+Datadog strongly recommends migrating to the new mappings as soon as possible before they become the default.
+</div>
+
+## Prerequisites
+
+Before migrating, remove any existing span name configuration:
+
+{{< tabs >}}
+{{% tab "OpenTelemetry Collector" %}}
+
+Remove `span_name_as_resource_name` and `span_name_remappings` from your Datadog exporter and connector configurations:
+
+{{< highlight py "hl_lines=4-6 11" >}}
+# Remove the highlighted lines if they exist in your configuration
+exporters:
+  datadog:
+    traces:
+      span_name_as_resource_name: true
+      span_name_remappings:
+        "old_name1": "new_name"
+
+connectors:
+  datadog/connector:
+    traces:
+      span_name_as_resource_name: true
+{{< /highlight >}}
+
+{{% /tab %}}
+{{% tab "Datadog Agent" %}}
+
+1. Remove `span_name_as_resource_name` and `span_name_remappings` from your Agent configuration:
+{{< highlight py "hl_lines=4-6" >}}
+# Remove the highlighted lines if they exist in your configuration
+otlp_config:
+  traces:
+    span_name_as_resource_name: true
+    span_name_remappings:
+      "old_name1": "new_name"
+{{< /highlight >}}
+
+2. Unset these environment variables:
+   - `DD_OTLP_CONFIG_TRACES_SPAN_NAME_AS_RESOURCE_NAME`
+   - `DD_OTLP_CONFIG_TRACES_SPAN_NAME_REMAPPINGS`
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Enable new operation name mappings
+
+{{< tabs >}}
+{{% tab "OpenTelemetry Collector" %}}
+
+Launch the OpenTelemetry Collector with the feature gate:
+
+```shell
+otelcol --config=config.yaml --feature-gates=datadog.EnableOperationAndResourceNameV2
+```
+
+### Replace removed configurations
+
+If you previously used span name configurations, replace them with processor configurations:
+
+#### Replace SpanNameAsResourceName
+
+The removed `span_name_as_resource_name` configuration pulled the `span.name` attribute from your OpenTelemetry trace to populate the Datadog operation name. To maintain this functionality, use a transform processor to map span names to the `operation.name` attribute:
```

Comment:
Initially I wasn't sure who or what removed the configuration mentioned here, and then I realized it was what we just told the user to do in the previous section. Again, probably my unfamiliarity, but maybe rephrasing could make it clearer, and a little more active?
```suggestion
The `span_name_as_resource_name` configuration you removed pulled the `span.name` attribute from your OpenTelemetry trace to populate the Datadog operation name. To maintain this functionality, use a transform processor to map span names to the `operation.name` attribute:
```